recipes/attributes-doc/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "attributes-doc" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/attributes-doc-{{ version }}.tar.gz
+  sha256: 1139b59601b4d71b90fa7681007160a048885e6d3c7c728ff26bc2cb46621d6a
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - typing  # [py<3]
```

Comment:
This package is pure python and should really be [noarch: python](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python). That means you can't use selectors.

Instead I'd just set `python >=3` in host and run requirements and remove the selector on `typing`.
Package won't be available on Python 2 but we don't build anymore for Python 2 on conda-forge anyway.